src/utils.js

```diff
@@ -36,6 +37,26 @@ let checkCurrentProject = () => {
     }
 }
 
+let checkLatestVersion = async () => {
+    try {
+        const response = await fetch(`https://registry.npmjs.org/${package.name}`);
```

Comment:
Thanks for sending this pull request @Sadaf-A. Could you please use `https.get` since we need to support old Node.js versions?